services/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str, password: str, email: str = None,
+                first_name: str = None, last_name: str = None) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
Consider handling the case where the User with the given 'user_id' does not exist, which would raise a 'DoesNotExist' exception. You might want to catch this exception and handle it appropriately.